python/src/worker/consume_message.py

```diff
@@ -97,11 +109,20 @@ def consume():
 
     if timeout <= datetime.datetime.utcnow():
         info(
-            f"Skipping processing task with uuid {mssg_body['uuid']}"
-            f"{mssg_body['uuid']} as its timeout of {timeout} has expired..."
+            f"Skipping processing task with ETag {mssg_body['ETag']} "
+            f"as its timeout of {timeout} has expired..."
```

Comment:
Why do we still have timeouts in the worker, if everything we compute will be used?